app/controllers/api/test_results_controller.rb

```diff
@@ -155,6 +142,13 @@ def update
       end
     end
 
+    # User params for create & update
+    def user_params
```

Comment:
We generally try to name this based on the model, so `test_result_params` rather than `user_params`.